packages/server-wallet/src/objectives/close-channel.ts

```diff
@@ -23,8 +23,9 @@ export class CloseChannelObjective {
               txSubmitterOrder: [1, 0],
```

Comment:
Side note: It seems like the `CloseChannelObjective` does not use the shared `insert` method? Do you know if there is a reason for that?

Reply:
Doesn't `CloseChannelObjective` use the same `ObjectiveModel.Insert` that other code uses?

Reply:
Yeah you are totally right. I misread this code.